src/components/Commits/CommitsListPage/CommitsListView.tsx

```diff
@@ -45,6 +76,19 @@ const CommitsListView: React.FC<React.PropsWithChildren<CommitsListViewProps>> =
     DEFAULT_VISIBLE_COMMIT_COLUMNS,
   );
   const [isColumnManagementOpen, setIsColumnManagementOpen] = React.useState(false);
+
+  const sortPaths: Record<SortableHeaders, string> = {
+    [SortableHeaders.name]: 'shaTitle',
+    [SortableHeaders.branch]: 'branch',
+    [SortableHeaders.byUser]: 'user',
+    [SortableHeaders.committedAt]: 'creationTime',
+    [SortableHeaders.status]: 'pipelineRuns[0].status.completionTime', // Status column uses custom priority-based sorting
```

Comment:
I'd use empty string `''` to avoid confusion, since we're not going to sort by `pipelineRuns[0].status.completionTime`